packages/app/src/AlertsPage.tsx

```diff
@@ -54,114 +90,78 @@ function disableAlert(alertId?: string) {
   // TODO do some lovely disabling of the alert here
 }
 
-function AlertDetails({
-  alert,
-  history,
-}: {
-  alert: AlertData;
-  history: AlertHistory[];
-}) {
+function AlertDetails({ alert }: { alert: AlertData }) {
   // TODO enable once disable handler is implemented above
   const showDisableButton = false;
+
   return (
-    <>
-      <div className="text-end">
+    <div className={styles.alertRow}>
+      <Group>
         {alert.state === AlertState.ALERT && (
-          <div className="badge bg-danger">ALERT</div>
-        )}
-        {alert.state === AlertState.OK && (
-          <div className="badge bg-success">OK</div>
+          <Badge color="red" size="sm">
+            Alert
+          </Badge>
         )}
+        {alert.state === AlertState.OK && <Badge size="sm">Ok</Badge>}
         {alert.state === AlertState.DISABLED && (
-          <div className="badge bg-secondary">DISABLED</div>
-        )}{' '}
+          <Badge color="gray" size="sm">
+            Disabled
+          </Badge>
+        )}
+
+        <Stack spacing={2}>
+          <div>
+            {alert.source === 'CHART' && alert.dashboard ? (
+              <Link href={`/dashboards/${alert.dashboard._id}`}>
+                <a className={styles.alertLink} title="Dashboard">
+                  <i className="bi bi-graph-up text-slate-400 me-2 fs-8" />
+                  {alert.dashboard.name}
```

Comment:
would be nice to surface chart name here (in addition to dashboard.name)

Reply:
I think we have information to pull out the chart name, right ? (dashboard record + chartId)